pr-dhl-woocommerce/includes/class-pr-dhl-wc-order-paket.php

```diff
@@ -1199,6 +1212,18 @@ public function modal_content_fields_pickup_request() {
 		}
 	}
 
+	protected function is_UK_shipment( $order_id ) {
+		$order = wc_get_order( $order_id );
+		$shipping_address = $order->get_address( 'shipping' );

```

Comment:
```suggestion
		$order = wc_get_order( $order_id );
		
		if ( ! is_a( $order, 'WC_Order' ) ) {
		    return false;
		}

		$shipping_address = $order->get_address( 'shipping' );

```
Let's make sure that the order id exists before moving forward.